_includes/footer.html

```diff
@@ -77,16 +77,14 @@ <h3 class="footer-heading">{{ site.title }}</h3>
 {% endif %}
 
 {% if site.ga_tracking_id %}
-<!-- Google Analytics -->
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id={{ site.ga_tracking_id }}"></script>
 <script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-  ga('create', '{{ site.ga_tracking_id }}', 'auto');
-  ga('send', 'pageview', {
```

Comment:
@johnnolan - any clue?

Reply:
I concur, I would add that snippet in just below the gtag config otherwise currently it wouldn't register (I believe anyway) the page view.

On another note, the google tag manager script could go in the header now as it is using the async attribute meaning the browser will fire off a background thread and not cause the page to pause rendering while it fetches the script.

Reply:
Thanks for this! Didn't see it was an async call! I've moved it into the head and added the page.url stuff back in.

Reply:
Thing of beauty, great stuff, looks all good from my end!